gitlab.go

```diff
@@ -469,3 +469,19 @@ func String(v string) *string {
 	*p = v
 	return p
 }
+
+// AccessLevelPtr is a helper routine that allocates a new AccessLevel value
+// to store v and returns a pointer to it.
+func AccessLevelPtr(v AccessLevel) *AccessLevel {
```

Comment:
Maybe we can rename the type `AccessLevel` to `AccessLevelValue` and rename this func to just `AccessLevel`?